apps/dashboard/src/contract-ui/tabs/overview/components/PermissionsTable.tsx

```diff
@@ -184,14 +183,7 @@ const PermissionsItem: React.FC<PermissionsItemProps> = ({ data }) => {
                 bg="transparent"
                 onClick={() => {
                   onCopy();
-                  toast({
-                    variant: "solid",
-                    position: "bottom",
-                    title: "Address copied.",
-                    status: "success",
-                    duration: 5000,
-                    isClosable: true,
-                  });
+                  toast.success("Address copied.");
```

Comment:
This shouldn't be "success" variant - if there is other visual feedback for copy action - we can completely drop this - otherwise use "info" variant